src/collections/Events.ts

```diff
@@ -43,5 +43,10 @@ export const Events: CollectionConfig = {
       type: "text",
       required: true,
     },
+    {
+      name: 'image',
```

Comment:
```suggestion
      name: 'heroImage',
```

Maybe we name this something more specific, like `heroImage` as we use probably quite a few different pictures for events.